JSONCoding/Classes/JSONUnarchiver.swift

```diff
@@ -157,17 +157,17 @@ open class JSONUnarchiver: JSONUnarchiving {
         }
     }
 
-    open static func topLevelUnarchived<T>(with rootJSON: Any, errorHandler: @escaping JSONUnarchiveErrorHandler) throws -> T where T: JSONCoding {
+    open class func topLevelUnarchived<T>(with rootJSON: Any, errorHandler: @escaping JSONUnarchiveErrorHandler) throws -> T where T: JSONCoding {
```

Comment:
This is a new warning generated by Xcode 10, you can't subclass `static` functions regardless of if they are `open/public`.